### PR TITLE
chore(css): Move CSS examples - imageset, fonts

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- part of https://github.com/mdn/mdn/issues/597

This PR moves some CSS external examples into content as part of an initiative to reduce repo maintenance and code duplication. 

/cc @bsmth

### CODE REVIEW

File: files/en-us/web/css/image/image-set/index.md [hunk 1]
```diff
@@ -77,22 +97,30 @@ This example shows how to use [`image-set()`](https://drafts.csswg.org/css-image
...
+      type("image/avif"),
+    "https://mdn.github.io/shared-assets/images/examples/balloons-landscape.jpg"
+      type("image/jpeg")
   );
 }
 ```
 
+{{EmbedLiveSample("image-set-type-example", "", "250px")}}
+
+{{EmbedGHLiveSample("css-examples/images/image-set-type.html", '100%', 600)}}
```
Comment (bsmth) [inline files/en-us/web/css/image/image-set/index.md:122]:
```suggestion
```

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.md [hunk 2]
```diff
@@ -124,28 +669,447 @@ Ordinals are also supported (such as '1st' or '3rd'), as is a slashed zero if pr
...
+```
+
+{{EmbedLiveSample("font-variant-numeric-frac-example", "", "550px")}}
 
 ### East Asian
 
 Associated CSS property: {{cssxref("font-variant-east-asian")}}
 
-This allows access to various alternate forms of glyphs within a font. The example below shows a string of glyphs with only the OpenType set 'jis78' enabled. Uncheck the box below and you'll see more characters displayed.
+This allows access to various alternate forms of glyphs within a font. The example below shows a string of glyphs with only the OpenType set 'jis78' enabled. Uncheck the box below and you'll see more characters displayed. Click "Play" in the code blocks below to edit the example in the MDN Playground:
```
Comment (bsmth) [inline files/en-us/web/css/css_fonts/opentype_fonts_guide/index.md:909]: Is this the inverse behavior? At first, I see two rows of characters (features active), then when I uncheck the box (the feature is inactive) and I see fewer characters. I think the prose is backwards, is that correct?

File: files/en-us/web/css/css_fonts/variable_fonts_guide/index.md [hunk 3]
```diff
@@ -316,9 +715,129 @@ h1 {
...
+  padding: 0;
+}
+
+.container * {
+  margin: 0.5rem auto 1rem;
+  max-width: 42rem;
+}
+```
+
+```css live-sample___sample-page-example
```
Comment (bsmth) [inline files/en-us/web/css/css_fonts/variable_fonts_guide/index.md:784]: I suppose we can combine these?
```suggestion

```

File: files/en-us/web/css/font-variant-numeric/index.md [hunk 4]
```diff
@@ -85,7 +85,29 @@ This property can take one of two forms:
...
+Click "Play" in the code blocks below to edit the example in the MDN Playground:
+
+```html live-sample___font-variant-numeric-example
+<p class="ordinal">1st, 2nd, 3rd, 4th, 5th</p>
+```
+
+```css live-sample___font-variant-numeric-example
+@font-face {
+  font-family: "Source Sans Pro";
+  src: url("https://mdn.github.io/shared-assets/fonts/SourceSansPro-Regular.otf")
```
Comment (bsmth) [inline files/en-us/web/css/font-variant-numeric/index.md:97]: I think this is a typo in the font URL. Should it be https://mdn.github.io/shared-assets/fonts/SourceCodePro-Regular.otf?

File: files/en-us/web/css/font-weight/index.md [hunk 5]
```diff
@@ -167,9 +167,32 @@ While many fonts have a particular weight corresponding to one of the numbers in
...
+```css live-sample___font-weight-example
+@font-face {
+  src: url("https://mdn.github.io/shared-assets/fonts/variable-fonts/MutatorSans.ttf");
+  font-family: "MutatorSans";
+  font-style: normal;
+  font-weight: 1 1000;
+}
+
+.sample {
+  text-transform: uppercase;
```
Comment (bsmth) [inline files/en-us/web/css/font-weight/index.md:188]:
```suggestion
  text-transform: uppercase;
  font-weight: 500;
```

File: files/en-us/web/css/font-weight/index.md [hunk 6]
```diff
@@ -167,9 +167,32 @@ While many fonts have a particular weight corresponding to one of the numbers in
 
 For TrueType or OpenType variable fonts, the "wght" variation is used to implement varying widths.
 
-This demo loads with `font-weight: 500;` set. Change the value of the `font-weight` property to see the weight of the text change.
+This demo loads with `font-weight: 500;` set. Change the value of the `font-weight` property to see the weight of the text change. Click "Play" in the code blocks below to edit the example in the MDN Playground:
```
Comment (bsmth) [inline files/en-us/web/css/font-weight/index.md:170]:
```suggestion
This demo loads with `font-weight: 500;` set. Change the value of the `font-weight` property in the `.sample` selector to see the weight of the text change (e.g., 200, 700). Click "Play" in the code blocks below to edit the example in the MDN Playground:
```

File: files/en-us/web/css/image/image-set/index.md [hunk 7]
```diff
@@ -68,7 +68,27 @@ Browsers do not provide any special information on background images to assistiv
...
+    url("https://mdn.github.io/shared-assets/images/examples/balloons-landscape.jpg")
+      2x
+  );
+}
+```
+
+{{EmbedLiveSample("image-set-example", "", "250px")}}
 
 > [!NOTE]
 > In the above example, the `-webkit` prefixed version is also used to support Chrome and Safari. In Firefox 90, support was added for `-webkit-image-set()` as an alias to `image-set()` (in order to provide compat where developers had not added the standard property).
```
Comment (bsmth) [inline files/en-us/web/css/image/image-set/index.md:94]: Is this gone now? Looks like it's not needed (the `-webkit` prefixed version)

File: files/en-us/web/css/font-variant-numeric/index.md (hunk 4 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/font-variant-numeric/index.md:97]: Ahh. The font file was among the HTML files, so it got skipped in the shared assets PR.  

Created a PR to commit it: https://github.com/mdn/shared-assets/pull/35

File: files/en-us/web/css/image/image-set/index.md (hunk 7 above)
Comment (OnkarRuikar) [inline files/en-us/web/css/image/image-set/index.md:94]:
```suggestion
```

File: files/en-us/web/css/css_fonts/opentype_fonts_guide/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/css_fonts/opentype_fonts_guide/index.md:909]: The prose say, "Uncheck the box below and you'll see more characters displayed." So current implementation is correct.

File: files/en-us/web/css/font-variant-numeric/index.md [hunk 8]
```diff
@@ -85,7 +85,29 @@ This property can take one of two forms:
...
+}
+
+.ordinal {
+  font-family: "Source Sans Pro";
+  font-size: 2rem;
+  font-variant-numeric: ordinal;
+}
+```
+
+{{EmbedLiveSample("font-variant-numeric-example", "", "200px")}}
```
Comment (bsmth) [inline files/en-us/web/css/font-variant-numeric/index.md:110]:
```suggestion
{{EmbedLiveSample("font-variant-numeric-example")}}
```

**Review by bsmth — APPROVED** [review]
Looking good, I think we can merge shortly!

**Pull request merged**